pkg/app/context.go

```diff
@@ -175,6 +175,10 @@ func (ctx *RequestContext) GetIndex() int8 {
 	return ctx.index
 }
 
+func (ctx *RequestContext) SetIndex(i int8) {
```

Comment:
Dangerous api: It's better not to offer a single one of this - doesn't make sense and just annoys the ordinary users

Reply:
thanks

Reply:
need an API interface to reset oldindex.